docs/src/developer/DEV_CONTAINER.md

```diff
@@ -1,86 +1,109 @@
 # VSCode Dev Containers
 
-VSCode Dev Containers are a great way to develop as they can contain all of the project's dependencies and in the end you have a read-to-use dockerFile which can also be used in your CI/CD pipeline.
+VSCode Dev Containers are great for developing as they contain all of the project's dependencies and in the end you have a ready-to-use dockerFile which can also be used in your CI/CD pipeline.
 
 It provides a normalized experience to all VSCode users across MacOS, Windows and Linux. All you need to get going is a the open source docker engine (not Docker Desktop!) and docker-compose.
```

Comment:
Technically it should be `the` as there is only one docker engine. And the `need not be` is not powerful enough for the user to prevent them using it.

Reply:
Agree on the `the` part.

But, why prevent anyone from using Docker desktop? It's **not recommended** in our context due to the recent licensing changes which impacted us as a group. But, for an OSS developer/enthusiast, "legally" using the free Docker desktop would not be an issue at all. He may even be using Docker desktop already. So, I don't see why we want to prevent anyone from using Docker desktop when the dev container setup doesn't care which flavour of docker you're using underneath (right?).

Reply:
I guess I don't want to writing instructions for people using docker desktop. Plus I feel that when people use docker desktop they think that everything should just "work"...which is definitely not the case, especially when you start at looking at volume mounting from the host filesystem (which is what most people on windows try to do straight away).